src/pages/Committees.js

```diff
@@ -140,15 +140,15 @@ function METAdorerna() {
     return(
         <p>
             METAdorerna (META doors) is the CS-Media committee that takes care of the Chapter Hall, META. If you do
-            not know yet, it is located at Osquars backe 21, 114 28 Stockholm, Sweden (or use this convenient 
+            not know yet, it is located at Osquars backe 21, 114 28 Stockholm, Sweden (or use this convenient&#20;
```

Comment:
Think usually people do `{' '}`, would that work? Might be easier for readers to understand, even if they don't know 20 is the code for space.

Up to you what you consider best, I don't insist

Reply:
Yeah, I agree :+1: